build_tools/before_all_mac.sh

```diff
@@ -1,4 +1,4 @@
-brew update
+#brew update
```

Comment:
The intention for this file is for it to run on the CI before building the wheels as part of the `cibuildwheel` step, could you revert this change?